### PR TITLE
WIP: Adding GraphqlType and SalesMan mongoose schema

### LINKED ISSUE #12: Criar  SalesMen model no mongoose
Criar a collection e o schema referente

### LINKED ISSUE #13: Salesmen type
Criar o Graphql Type expondo os dados para somente os usuários que estão logados

### PR DESCRIPTION
**Pull request by tiagoheliob**
closes #12

### CODE REVIEW

File: src/model/SalesMen.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+
+const SalesMenSchema = new mongoose.Schema(
```
Comment (renatobenks) [inline src/model/SalesMen.js]: Your model don't has the sales field?

Comment (tiagoheliob) [reply src/model/SalesMen.js]: Yes, but I Don't how i can represent it still.

Comment (renatobenks) [reply src/model/SalesMen.js]: all right, so search for http://mongoosejs.com/docs/guide.html

File: src/model/SalesMen.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  {
+    name: {
+      type: String,
+    },
+    birth_day: {
```
Comment (renatobenks) [inline src/model/SalesMen.js]: it's a only one word.

File: src/model/SalesMen.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      type: String,
+    },
+    hired_date: {
```
Comment (renatobenks) [inline src/model/SalesMen.js]: Use snakeCase it's better to define this, please, `hiredDate`.

File: src/model/SalesMen.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import mongoose from 'mongoose';
+
+const SalesMenSchema = new mongoose.Schema(
```
Comment (renatobenks) [inline src/model/SalesMen.js]: No one about this fields are required?

File: src/model/SalesMen.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        vehicle: {
+          type: Schema.ObjectId,
+          ref: 'Vehicle',
+        },
+        commission_amount: Number,
+      },
+    ],
+  },
+  {
+    timestamps: {
```
Comment (renatobenks) [inline src/model/SalesMen.js]: Why this? Will gonna be utility for your collection?

File: src/model/SalesMen.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      type: Number,
+      required: true,
+    },
+    sales: [
+      {
+        vehicle: {
+          type: Schema.ObjectId,
+          ref: 'Vehicle',
+        },
+        commission_amount: Number,
```
Comment (renatobenks) [inline src/model/SalesMen.js]: What you want represents in this field?

Comment (tiagoheliob) [reply src/model/SalesMen.js]: I'll delete it because the commission amount is a vehicle attribute

**Pull request merged**